### PR TITLE
#17 Resolve task Delete Question

### PR DESCRIPTION
**Pull request by HaiTranVar**

### CODE REVIEW

File: app/views/tests/show.html.erb [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
-          <li><%= answer.content %></li>
-        <% end %>
+<h2 class="text-center"><%= @test.name %></h2>
+<h4 class="text-right"><%= t ".times" %><%= @test.time %></h4>
+<h4 class="text-right"><%= link_to t(".back"), root_path %></h4>
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th>Question ID</th>
+      <th class="text_center">Question</th>
```
Comment (HaiTranVar) [inline app/views/tests/show.html.erb]: i18n

File: app/controllers/questions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class QuestionsController < ApplicationController
+  before_action :correct_question,   only: :destroy
```
Comment (ducminhv) [inline app/controllers/questions_controller.rb]: one space plz

Comment (ducminhv) [reply app/controllers/questions_controller.rb]: add blank line after this line

File: app/controllers/questions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  def destroy
+    @question.destroy
+    flash[:success] = "Question deleted"
+    redirect_to test_path @question.test
+  end
```
Comment (ducminhv) [inline app/controllers/questions_controller.rb]: add blank line after this line

File: app/controllers/questions_controller.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class QuestionsController < ApplicationController
+  before_action :correct_question,   only: :destroy
+  def destroy
+    @question.destroy
+    flash[:success] = "Question deleted"
```
Comment (ducminhv) [inline app/controllers/questions_controller.rb]: i18n

File: app/views/tests/show.html.erb [hunk 5]
```diff
@@ -1,19 +1,42 @@
...
+<h4 class="text-right"><%= t ".times" %><%= @test.time %></h4>
+<h4 class="text-right"><%= link_to t(".back"), root_path %></h4>
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th>t ".question_id"</th>
+      <th class="text_center">t ".question"</th>
+      <th class="text-center">t ".edit"</th>
+      <th class="text-center">t ".delete"</th>
+
```
Comment (ducminhv) [inline app/views/tests/show.html.erb]: remove blank line

**Pull request merged**